mapactionpy_arcmap/arcmap_runner.py

```diff
@@ -29,41 +26,50 @@ def is_valid_file(parser, arg):
     else:
         return arg
 
+
 def is_valid_directory(parser, arg):
     if os.path.isdir(arg):
         return arg
     else:
         parser.error("The directory %s does not exist!" % arg)
         return False
 
+
 def main(args):
     args = parser.parse_args()
-    cookbookFile=args.cookbookFile
-    layerPropertiesFile=args.layerConfig    
-    mxdTemplate=args.templateFile    
-    crashMoveFolder=args.crashMoveFolder
-    layerDirectory=args.layerDirectory
-    productName=args.productName
-    countryName=args.countryName
+    cookbookFile = args.cookbookFile
+    layerPropertiesFile = args.layerConfig
+    mxdTemplate = args.templateFile
+    crashMoveFolder = args.crashMoveFolder
+    layerDirectory = args.layerDirectory
+    productName = args.productName
+    countryName = args.countryName
 
     mxd = arcpy.mapping.MapDocument(mxdTemplate)
 
-    chef = MapChef(mxd, cookbookFile, layerPropertiesFile, crashMoveFolder, layerDirectory)    
+    chef = MapChef(mxd, cookbookFile, layerPropertiesFile, crashMoveFolder, layerDirectory)
     chef.cook(productName, countryName)
     reportJson = chef.report()
+    print(reportJson)
+
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
         description='This component accepts a template MXD file, a list of the'
         'relevant datasets along with other information required to create an'
         'event specific instance of a map.',
     )
-    parser.add_argument("-b", "--cookbook", dest="cookbookFile", required=True, help="path to cookbook json file", metavar="FILE", type=lambda x: is_valid_file(parser, x))
-    parser.add_argument("-l", "--layerConfig", dest="layerConfig", required=True, help="path to layer config json file", metavar="FILE", type=lambda x: is_valid_file(parser, x)) 
-    parser.add_argument("-t", "--template", dest="templateFile", required=True, help="path to MXD file", metavar="FILE", type=lambda x: is_valid_file(parser, x)) 
-    parser.add_argument("-cmf", "--cmf", dest="crashMoveFolder", required=True, help="path the Crash Move Folder", metavar="FILE", type=lambda x: is_valid_directory(parser, x)) 
-    parser.add_argument("-ld", "--layerDirectory", dest="layerDirectory", required=True, help="path to layer directory", metavar="FILE", type=lambda x: is_valid_directory(parser, x)) 
-    parser.add_argument("-p", "--product", dest="productName", required=True, help="Name of product") 
-    parser.add_argument("-c", "--country", dest="countryName", required=True, help="Name of country") 
+    parser.add_argument("-b", "--cookbook", dest="cookbookFile", required=True,
```

Comment:
Low priority but in general I think that may be ways to reduce the number of parameters that user has to supply having them pass the path to the cmf_config_file and then using a CrashMoveFolder object to obtain the other parameters.